apps/web/test/lib/getSchedule.test.ts

```diff
@@ -86,6 +88,287 @@ describe("getSchedule", () => {
     });
   });
 
+  describe("Round robin lead skip - CRM", async () => {
+    test("correctly get slots for event with only round robin hosts", async () => {
+      const { dateString: plus1DateString } = getDate({ dateIncrement: 1 });
+      const { dateString: plus2DateString } = getDate({ dateIncrement: 2 });
```

Comment:
Saw in @hariombalhara recent PR #15155 that getDate doesn't take into account timezone data. Will this affect this test on month change or DST changes?

https://github.com/calcom/cal.com/blob/26bd02bd9409123f50f3501471738da66554f558/apps/web/test/utils/bookingScenario/bookingScenario.ts#L656-L659